dev-support/Jenkinsfile

```diff
@@ -40,7 +40,7 @@ pipeline {
     OUTPUT_DIR_RELATIVE_JDK11_HADOOP3 = 'output-jdk11-hadoop3'
 
     PROJECT = 'hbase'
-    PROJECT_PERSONALITY = 'https://raw.githubusercontent.com/apache/hbase/master/dev-support/hbase-personality.sh'
+    PROJECT_PERSONALITY = 'https://raw.githubusercontent.com/apache/hbase/HBASE-24198/dev-support/hbase-personality.sh'
```

Comment:
This correct? For master branch?

Reply:
This is just for verifying. As I started a nightly job for HBASE-24198, where I changed the personality but here in Jenkinsfile we still reference the one from master branch...

Will revert this change when merging.